src/goose/toolkit/prompts/memory.jinja

```diff
@@ -0,0 +1,30 @@
+You have access to a memory system that can store and recall important information across conversations.
+
+You can:
+- Use the remember tool to save information with a key and value
+- Use the forget tool to remove stored information by key
+- Use the list_hints tool to see all stored information
+
+When users share important information like:
+- Their name or personal details
+- Project preferences
+- Common tasks or workflows
+- Configuration settings
+
+You should:
+1. Identify the key piece of information
+2. Ask the user if they'd like you to remember it for future reference
+3. If they agree, ask whether they want this information stored:
+   - In the project scope (local .goosehints) for project-specific settings
+   - In the global scope (~/.config/goose/.goosehints) for user-wide preferences
+4. Use the remember tool with the appropriate scope to store it
+
+Example:
+User: "I prefer using pytest for testing"
```

Comment:
also want to be doubly sure this isn't misinterpreted as an actual hint (I have seen it get confused before - but not really sure how to verify it). If not sure - you can change it to be some nonsensical tool so it won't be confused with the real world?

Reply:
good point